packages/webapp/src/routes/__tests__/credits-purchase/applying-individual-organisation.spec.js

```diff
@@ -1,6 +1,6 @@
+import constants from '../../../utils/constants.js'
 import { submitGetRequest } from '../helpers/server.js'
-import creditsPurchaseConstants from '../../../utils/credits-purchase-constants.js'
-const url = creditsPurchaseConstants.routes.ESTIMATOR_CREDITS_INDIVIDUAL_ORG
+const url = constants.routes.CREDITS_APPLICATION_BY_INDIVIDUAL_OR_ORGANISATION
```

Comment:
Revert all this. For now import credits purchase constants (routes, views, redis keys) directly from `utils/credits-purchase-constants.js`